packages/stark-ui/src/modules/minimap/components/minimap.component.ts

```diff
@@ -12,7 +12,9 @@ import {
 	Renderer2,
 	ViewEncapsulation
 } from "@angular/core";
-import { ItemVisibility, StarkDOMUtil, StarkMinimapItemProperties } from "@nationalbankbelgium/stark-ui";
+
+import { ItemVisibility, StarkMinimapItemProperties } from "../../minimap";
```

Comment:
Same remark, imports should be refined like this:

```typescript
import { ItemVisibility } from "../item-visibility.intf";
import { StarkMinimapItemProperties } from "../item-properties.intf";
```

Reply:
Good catch 😉